package.json

```diff
@@ -102,9 +102,15 @@
             "type": "string"
           },
           "default": [
+            "::",
+            "--",
+            "//",
+            "@@",
             ":",
             "-",
-            "/"
+            "/",
+            "@"
```

Comment:
Can you provide references to these new defaults where they have been used?

Reply:
Tbh the only refenced I had was "--" because of [@iconify/tailwind](https://github.com/iconify/iconify/tree/main/plugins/tailwind).

I thought those could be useful as well for custom sets. although it's not necessary since I changed some logics to support multi characters anyways.

if it's needed, I can make another pr with only "--" preset.

Reply:
Yes please update this PR, let's only have the defaults that are necessary

Reply:
@antfu it's done